client/src/components/Announcements.js

```diff
@@ -10,13 +15,61 @@ class AnnouncementList extends Component {
     constructor(props) {
       super(props);
       this.state = {
-        announcements : this.props.announcements,
+        announcements : [],
           id: this.props.id
       }
+      
+    }
+
+    /**
+     * Makes a GET request to either dev or prod and fetches the announcement list
+     * as soon as the component loads and updates the state
+     * TODO: Add Validation for output
+     * TODO: Once finished, remove debugging statements
+     */
+    componentDidMount() {
+        if (this.ReactIsInDevelomentMode()) {
+            let urlToFetch = API_ANNOUNCEMENTS_DEV;
+            fetch(urlToFetch)
+            .then(response => response.json())
+            .then(data =>  this.setState({announcements:this.parseAnnouncements(data)}))
+          } else {
+            let urlToFetch = API_ANNOUNCEMENTS_DEV;
+            fetch(urlToFetch)
+            .then(response => response.json())
+            .then(data =>  this.setState({announcements:this.parseAnnouncements(data)}))
+          }
     }
 
+    /**
+     * TODO: How are multiple announcements stored? in data.annoucnment or just as a string
```

Comment:
Here's what it looks like:
https://github.com/GTBitsOfGood/bright-futures-academy/pull/83

Let me know if you need some help with making the calls